### DISCUSSION TITLE
Найцікавіші новини місяця: листопад 2023

### DISCUSSION THREAD

**Discussion by rkiyanchuk** [shopokodu/community#40] [Теми для подкасту]
**Теми ведучих:**

- [x] В Python [нарешті буде валідація сертифікатів](https://github.com/pyca/cryptography/issues/2381#issuecomment-1816822268) 
- [x] Go [виповнилось 14 років](https://go.dev/blog/14years).
- [x] [Notepad++: 20th-Year Anniversary](https://notepad-plus-plus.org/news/v86-20thyearanniversary/)
- [x] [Reflecting on 18 years at Google](https://ln.hixie.ch/?start=1700627373&count=1)
- [x] [Python is Easy. Go is Simple. Simple != Easy.](https://preslav.me/2023/11/27/python-is-easy-golang-is-simple-simple-is-not-easy/)
- [x] [What Linux looked like in the year 1998](https://scorpioncity.com/lishots.html)

**Теми случахів:**

- [x] [Ampere's 192-core chips break ARM64 Linux kernel in two-socket systems, company requests higher core count support](https://www.tomshardware.com/pc-components/cpus/yes-you-can-have-too-many-cores-amperes-192-core-cpus-break-arm64-linux-kernel-in-two-socket-systems-company-requests-higher-core-count-support-for-mainline-linux)
- [x] [Turbo Pascal turns 40](https://blogs.embarcadero.com/turbo-pascal-turns-40/)
- [x] [Advent of Code](https://adventofcode.com/)
- [x] [Wine Wayland Driver's Vulkan Support Is Now Usable](https://www.phoronix.com/news/Wine-Wayland-Vulkan-Usable)
- [x] [/e/OS 1.17!](https://e.foundation/about-e/)

**Comment by sdongles** [comment]
[Ampere's 192-core chips break ARM64 Linux kernel in two-socket systems, company requests higher core count support](https://www.tomshardware.com/pc-components/cpus/yes-you-can-have-too-many-cores-amperes-192-core-cpus-break-arm64-linux-kernel-in-two-socket-systems-company-requests-higher-core-count-support-for-mainline-linux)
